src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/SpotifyWidgetPageObject.java

```diff
@@ -11,12 +13,17 @@
 
   @FindBy(css = "iframe[data-wikia-widget='spotify']")
   private WebElement spotifyIframe;
+  @FindBy(css = "iframe[data-wikia-widget='spotify']")
+  private List<WebElement> spotifyIframeList;
   @FindBy(css = "#widgetContainer")
   private WebElement spotifyBody;
 
   private static final String TAG_NAME = "spotify";
   private static final String ARTICLE_NAME = "spotifyWidget";
-  private static final String TAG = "<spotify uri=\"spotify:track:5JunxkcjfCYcY7xJ29tLai\" />";
+  private static final String TAGS[] = {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)